java/dataset/src/test/java/org/apache/arrow/dataset/TestDataset.java

```diff
@@ -38,9 +40,13 @@
 import org.apache.arrow.vector.ipc.ArrowReader;
 import org.apache.arrow.vector.ipc.message.ArrowRecordBatch;
 import org.apache.arrow.vector.types.pojo.Schema;
+import org.apache.calcite.sql.parser.SqlParseException;
 import org.junit.After;
 import org.junit.Before;
 
+import io.substrait.isthmus.SqlToSubstrait;
```

Comment:
I don't think it really matters to use Isthmus here. Just check in the generated plan and drop the dependency.

Reply:
If testing with Isthmus specifically is a big deal, we should consider how to set up specific integration tests with important third party dependencies/dependents.